metrics.go

```diff
@@ -14,11 +14,18 @@ import (
 	"github.com/pkg/errors"
 )
 
-const maxLabelValueLength = 80
+const (
+	maxLabelValueLength = 80
+	geo_lat             = "lat"
+	geo_lon             = "lon"
+	geo_hash            = "geo_hash"
```

Comment:
Golang naming convention would be `GeoLat,` `GeoLon`, `GeoHash` IIRC

Reply:
True, was thinking more along the lines of metrics naming.